src/index.ts

```diff
@@ -93,9 +98,11 @@ export function apply(ctx: Context, config: Config) {
 
       try {
         // send a message and wait for the response
-        const { conversationId, messageId } = conversations.get(key) ?? {}
-        const response = await api.sendMessage({ message: input, conversationId, messageId })
-        conversations.set(key, { conversationId: response.conversationId, messageId: response.messageId })
+        let request: Conversation = conversations.get(key)
+        let action: ChatGPT.Action = input in actions ? actions[input] : 'next'
```

Comment:
Also always use `const`, unless you really really need to change it in runtime, otherwise keep it immutable.